providers/nitro/nitro-helper/Cargo.toml

```diff
@@ -28,7 +28,7 @@ tempfile = "3"
 tendermint = { version = "0.19" }
 thiserror = "1"
 tmkms-light = { path = "../../.." }
-tokio = { version = "= 0.2", features = [ "rt-core", "time" ] }
+tokio = { version = "1.6", features = [ "rt", "rt-multi-thread" ] }
```

Comment:
is `rt-multi-thread` needed, given only one call is made? perhaps `Runtime::Builder::new_current_thread()` is enough